content/en/docs/concepts/services-networking/service.md

```diff
@@ -432,7 +433,7 @@ You can find more information about `ExternalName` resolution in
 ## Headless Services
 
 Sometimes you don't need load-balancing and a single Service IP.  In
-this case, you can create what are termed “headless” Services, by explicitly
+this case, you can create what is termed “headless” Services, by explicitly
```

Comment:
This is already correct, no change required. 👎

Reply:
okay